modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -27,23 +25,24 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-ibm-power"]
 :ibm-power:
 endif::[]
-ifeval::["{context}" == "installing-platform-agnostic"]
-:agnostic:
+ifeval::["{context}" == "installing-rhv-restricted-network"]
+:rhv:
```

Comment:
I assume this attribute should not be replacing the platform agnostic attribute? If not, can you add the RHV attribute as a new one and keep the agnostic attributes and text the same in this module?